src/Service/GithubServiceTrait.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/).
+ *
+ * @link      http://github.com/zendframework/zend-version for the canonical source repository
+ * @copyright Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+
+namespace Zend\Version\Service;
+
+use Zend\Json\Json;
+
+/**
+ * Allows Github's git/refs api to be parsed.
+ *
+ * @see http://developer.github.com/v3/git/refs/#get-all-references
+ */
+trait GithubServiceTrait
```

Comment:
I don't care much for traits either.  So, I debated about the implementation.  This seemed to be the most efficient, as opposed to `Github\StreamService` and `Github\ClientService`, especially since this is geared towards user apps, not the framework.  See the PR example.